addons/sale_timesheet/tests/test_sale_timesheet.py

```diff
@@ -526,6 +526,7 @@ def test_timesheet_invoice(self):
 
     def test_transfert_project(self):
         """ Transfert task with timesheet to another project. """
+        self.env.user.employee_id = self.env['hr.employee'].create({'user_id': self.env.uid})
```

Comment:
I thought the current user has an employee. 🤔

Reply:
@xavierbol  Not in this case; the test class `TestSaleTimesheet` is inheriting from  `TestCommonSaleTimesheet -> TestSaleCommon -> AccountTestInvoicingCommon` (that create the user set to env) but none of them creates an employee for the `self.env.user`:
https://github.com/odoo/odoo/blob/1e69c4fe5f8dd8a92d92f350b6d6a9539157f206/addons/account/tests/common.py#L39-L44